samples/Client/DaprClient/DaprClient.csproj

```diff
@@ -2,7 +2,9 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net5</TargetFramework>
```

Comment:
I thought we are going to target net5 only for the tests. The other samples are also targeting 3.1

Reply:
Good point, I'll revert this part.